src/main/java/org/traccar/protocol/MobilogixProtocolEncoder.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2025 Joaquim Cardeira (joaquim@cardeira.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.traccar.protocol;
+
+import org.traccar.Protocol;
+import org.traccar.StringProtocolEncoder;
+import org.traccar.helper.DateUtil;
+import org.traccar.model.Command;
+
+import java.util.Date;
+
+public class MobilogixProtocolEncoder extends StringProtocolEncoder {
+    public MobilogixProtocolEncoder(Protocol protocol) {
+        super(protocol);
+    }
+
+    private Object encodeCommand(Date time, String param) {
+        return String.format("[%s,S6,%s]", DateUtil.formatDate(time, false), param);
+    }
+
+    @Override
+    protected Object encodeCommand(Command command) {
+        return encodeCommand(command, new Date());
+    }
+
+    protected Object encodeCommand(Command command, Date time) {
```

Comment:
Why do we need a separate method with protected modifier?

Reply:
For the test

Reply:
I guess that makes sense.